cmd/manager/aggregator.go

```diff
@@ -509,6 +509,7 @@ func getRemediationLabels(scan *compv1alpha1.ComplianceScan, obj runtime.Object)
 func getCheckResultLabels(pr *utils.ParseResult, resultLabels map[string]string, scan *compv1alpha1.ComplianceScan) map[string]string {
 	labels := make(map[string]string)
 	labels[compv1alpha1.ComplianceScanLabel] = scan.Name
+	labels[compv1alpha1.ProfileGuidLabel] = scan.Labels[compv1alpha1.ProfileGuidLabel]
```

Comment:
I wonder if we should change this label to be more specific, so that it's not as easy for someone to interpret this as being the GUID for the CCR:

```
╭─lbragstad@p1 ~
╰─➤  $ oc get ccr  -o json | jq '.items[].metadata.labels."compliance.openshift.io/guid"' 
 "97265368-738f-5e81-8475-03cee08b1af5"
```
`"compliance.openshift.io/profile-guid"` would make it more clear that the ID-looking-thing here maps to the profile, not the ComplianceCheckResult or ComplianceScan.

Reply:
agree, I think that's a good point